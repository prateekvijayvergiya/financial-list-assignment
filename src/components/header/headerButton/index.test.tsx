import Component from './'
import makeSnapshot from '../../../common/testing/makeSnapshot'

describe('HeaderButton', () => {
  it('renders correctly', () => {
    const props = {
      title: 'Price',
      handleSortChange: jest.fn(),
      sortKey: 'price',
      sortConfig: null,
    }
    makeSnapshot(<Component {...props} />)
  })
})
