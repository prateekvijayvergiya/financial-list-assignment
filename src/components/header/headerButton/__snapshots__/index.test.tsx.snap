// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderButton renders correctly 1`] = `
<th
  className="sc-bdnxRM cnKWit"
  onClick={[Function]}
>
  <button
    className="sc-gtsrHT ddXyUD"
  >
    Price
  </button>
</th>
`;
